src_tests/integration_tests/use_case_tests/test_usecase_22_export.py

```diff
@@ -0,0 +1,15 @@
+import os
+from pathlib import Path
+
+from src.osm_configurator.model.project.active_project import ActiveProject
+from src_tests.definitions import TEST_DIR
+import src.osm_configurator.model.application.application_settings as application_settings_i
+
+
+class TestUseCase22:
```

Comment:
There is an Asserton in this test, but it is not really meaningful. It is only tested if the return value is correct. I would add an Assertion that tests whether the file was actually created